crates/bdk/src/wallet/signer.rs

```diff
@@ -162,16 +163,8 @@ pub enum SignerError {
     SighashError(sighash::Error),
     /// Miniscript PSBT error
     MiniscriptPsbt(MiniscriptPsbtError),
-    /// Error while signing using hardware wallets
-    #[cfg(feature = "hardware-signer")]
-    HWIError(hwi::error::Error),
-}
-
-#[cfg(feature = "hardware-signer")]
-impl From<hwi::error::Error> for SignerError {
-    fn from(e: hwi::error::Error) -> Self {
-        SignerError::HWIError(e)
-    }
+    /// A generic error
+    Generic(String),
```

Comment:
I'm not a big fan of the `SignerError::Generic` since I just removed `bdk::Error::Generic` in #1028. But in this case I don't see an simple way around it. If it becomes annoying we can fix it in 2.0 release.